components/core/UserManager.py

```diff
@@ -38,9 +37,9 @@ def check_user_name(username):
 def get_user(username):
     db = threadpool.connect()
     if db is not None:
-        cursor = db.cursor(MySQLdb.cursors.DictCursor)
+        cursor = db.cursor()#MySQLdb.cursors.DictCursor)
         sql = "SELECT * FROM user WHERE user_name='%s' and blocked=0;" % username
-        cursor.execute(sql)
+        cursor.execute(sql) 
```

Comment:
If you remove the tailing whitespace here, Codacy linting will pass. 

https://www.codacy.com/app/agentmilindu/Bassa/pullRequest?prid=476397#